lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,12 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        languages = settings.CERTIFICATE_TEMPLATE_LANGUAGES
+        lang_choices = [(key, languages[key]) for key in languages]
```

Comment:
Could you just use `languages.items()` (https://docs.python.org/2/library/stdtypes.html#dict.items)? Also, you also might want to sort these as dictionaries do not guarantee order.